specs/Caesar.cry

```diff
@@ -0,0 +1,38 @@
+/* Simple Caesar cipher (ROT13) */
+module Caesar where
```

Comment:
I imagine `module specs::Caesar` would be better, just in case some ancillary files elsewhere (such as examples in the labs) want to reference modules from here.

Reply:
Hmm...that would be amusing: a module collecting all the tutorial ciphers.  I'll start by mirroring the organization of [cryptol-specs](https://github.com/GaloisInc/cryptol-specs), under which Caesar is a symmetric block cipher...